docs/ICON_DESIGN_GUIDE.md

```diff
@@ -0,0 +1,78 @@
+# Icon Degin Guide
+
+To design icons for this icon library we need to follow some guidelines to match all the icons in the same style.
+
+## Summary of the rules we have
+
+1. Each Icon is desined on a canvas of **24 by 24 pixels**.
+2. Each Icon have **1 pixel padding** between icon and the canvas.
+3. Each icon has a **stroke width of 2 pixels**.
+4. Each icon has **round joins**.
+5. Each icon has **round caps**.
+6. Each icon has **centered strokes**.
+7. Each icon containing squares has a **border radius of 2 pixels**.
+8. Each icon **2 pixels of spacing between elements**.
+
+## The Rules Visualized
```

Comment:
I'd remove that and directly have the rules.

Reply:
Yeah with the images, it not directly clear what the rules are. So thats why I added a summary